CHANGELOG.md

```diff
@@ -5,6 +5,193 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
 tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.3](https://github.com/oscar-system/Oscar.jl/releases/tag/v1.2.3) - 2025-02-28
+
+The following gives an overview of the changes compared to the previous release. This list is not
+complete, many more internal or minor changes were made, but we tried to only list those changes
+which we think might affect some users directly.
+
+### Highlights
+
+- [#4399](https://github.com/oscar-system/Oscar.jl/pull/4399) Graduate Weyl groups and root systems from experimental to officially supported
+- [#4294](https://github.com/oscar-system/Oscar.jl/pull/4294) Graduate elliptic surfaces from experimental
+
+### Renamings
+
+- [#4526](https://github.com/oscar-system/Oscar.jl/pull/4526) Rename `blowup` and `blow_up_points`  to ` blow_up` and `blow_up_points`, and other intersection theory changes
+- [#4433](https://github.com/oscar-system/Oscar.jl/pull/4433) Rename `new_ray` to `exceptional_ray`
+- [#4304](https://github.com/oscar-system/Oscar.jl/pull/4304) Change `hom` to `map` in in intersection theory
+- [#4302](https://github.com/oscar-system/Oscar.jl/pull/4302) Rename `acting_domain` to `acting_group` and some improvements for group cosets
+- [#4278](https://github.com/oscar-system/Oscar.jl/pull/4278) Rename `minimal_generating_set` for groups to `minimal_size_generating_set`
+
+### Algebraic Geometry
+
+- [#4615](https://github.com/oscar-system/Oscar.jl/pull/4615) Speed up computations for EllipticSurface by reduction to positive characteristic.
+- [#4560](https://github.com/oscar-system/Oscar.jl/pull/4560) Add a database of Enriques surfaces
+- [#4540](https://github.com/oscar-system/Oscar.jl/pull/4540) Support for computing automorphism groups of Enriques surfaces
+- [#4534](https://github.com/oscar-system/Oscar.jl/pull/4534) Add `stabilizer_in_orthogonal_group` to compute stabilisers of timelike vectors
+- [#4485](https://github.com/oscar-system/Oscar.jl/pull/4485) Make `saturation` for principal ideals faster by delegating to `remove`
+- [#4452](https://github.com/oscar-system/Oscar.jl/pull/4452) Add `graph_curve(::Graph)`
```

Comment:
```suggestion
```
This was merged after the `release-1.3` branch was created and has no backport label